src/System.ServiceModel.NetTcp/src/System.ServiceModel.NetTcp.csproj

```diff
@@ -0,0 +1,58 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AssemblyVersion>$(WcfAssemblyVersion)</AssemblyVersion>
+    <StrongNameKeyId>Microsoft</StrongNameKeyId>
+    <AssemblyName>System.ServiceModel.NetTcp</AssemblyName>
+    <CLSCompliant>true</CLSCompliant>
+    <IsPackable>true</IsPackable>
+    <!-- Workaround for https://github.com/NuGet/NuGet.Client/pull/3016 -->
+    <NoWarn>$(NoWarn);NU5131</NoWarn>
+    <TargetFrameworks>net6.0;net461</TargetFrameworks>
```

Comment:
It looks to me like your `net461` implementation is a full facade.  Since you aren't exposing any types on that `TargetFramework` and you don't have a `netstandard2.0` assembly with types that you need to replace with forwards on .NETFramework , could you just delete your `net461` build?